swaglyrics/spotify.py

```diff
@@ -1,7 +1,8 @@
-import win32gui
-
+import platform
 
 def get_info_windows():
+	import win32gui
```

Comment:
won't this give an error on linux?

Reply:
we are not calling this function in linux. That's why I imported inside the function